src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -3,6 +3,8 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import type { Multer } from 'multer';
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/47780
@acd124 help please

Reply:
This may have to do with `"types": ["vitest/globals"]` in the tsconfig. further reading into comments around here https://github.com/DefinitelyTyped/DefinitelyTyped/issues/47780#issuecomment-854603627 indicated that there are various methods to make this work, with different upsides and downsides. I added "Multer" to the types array and it did fix the prisma:reset but broke yarn start. I tried removing the line entirely and that seemed to fix the issue. (also removed this addition of unused import).

Reply:
removed import and tsconfig types array